hoodie-cli/src/main/java/com/uber/hoodie/cli/commands/HoodieLogFileCommand.java

```diff
@@ -172,7 +175,8 @@ public String showLogFileRecords(@CliOption(key = {
               .getTimestamp(),
           Long.valueOf(HoodieMemoryConfig.DEFAULT_MAX_MEMORY_FOR_SPILLABLE_MAP_IN_BYTES),
           Boolean.valueOf(HoodieCompactionConfig.DEFAULT_COMPACTION_LAZY_BLOCK_READ_ENABLED),
-          Boolean.valueOf(HoodieCompactionConfig.DEFAULT_COMPACTION_REVERSE_LOG_READ_ENABLED));
+          Boolean.valueOf(HoodieCompactionConfig.DEFAULT_COMPACTION_REVERSE_LOG_READ_ENABLED),
+          Integer.valueOf(HoodieMemoryConfig.DEFAULT_MAX_HDFS_STREAM_BUFFER_SIZE));
```

Comment:
rename to : `..MAX_DFS_STREAM_BUFFER_SIZE`

Reply:
done